src/webgpu/shader/execution/expression/call/builtin/acosh.spec.ts

```diff
@@ -31,7 +36,14 @@ g.test('f32')
   .params(u =>
     u.combine('inputSource', allInputSources).combine('vectorize', [undefined, 2, 3, 4] as const)
   )
-  .unimplemented();
+  .fn(async t => {
+    const makeCase = (n: number): Case => {
+      return makeUnaryF32IntervalCase(n, ...acoshIntervals);
+    };
+
+    const cases = fullF32Range().map(makeCase);
```

Comment:
I'd feel better if there was a relatively dense set of points around the "hard" part of the function: at x=1 and just above 1.

Reply:
Added some additional test cases near x = 1

Reply:
I used a biased interval of [1, 2] with 100 samples for the additional test cases, I can easily change the bounds if you want.